form.go

```diff
@@ -59,6 +59,9 @@ type Form struct {
 	// to be more accessible to screen readers.
 	accessible bool
 
+	// the program this form is running in, used for testing
+	program *tea.Program
```

Comment:
Can we keep the program out of the form. We can send an `Update` to the form which allows the same keypress to be sent during testing.

Reply:
The handler for `keymap.Quit` then returns a `tea.Quit` cmd to exit the program, but that won't happen if I use something like `batchUpdate`. What would the correct way be to handle this?

Reply:
A form is actually simply a bubble tea model, so we can create a new tea program and just run that here and pass the form model to the program.

You would also need to set the quitCmd etc...

Reply:
I see, that works. However, the intention was to test the `Run()` method, because that's where the issue was. Creating a new program inside the test would bypass the `Run()` method, no?